content/client-lib-development-guide/features.textile

```diff
@@ -1605,7 +1607,7 @@ class RealtimeClient: // RTC*
   constructor(keyOrTokenStr: String) // RTC12
   constructor(ClientOptions) // RTC12
   auth: Auth // RTC4
-  push: Push
+  push: Push // RTC13
```

Comment:
Confusing with RSC21 above, maybe to add ref to Push object itself here as well?

Reply:
I've meant that both of them are the same Push object, but have a different spec point, which was confusing. But that's how it should be, so nevermind =)